docs/services/data/capture-sync.md

```diff
@@ -614,52 +714,184 @@ The following example captures data from the `ReadImage` method of a camera:
 
-{{<imgproc src="/services/data/data-icon.png" resize="300x" declaredimensions=true alt="Data icon">}}
+To view all the captured data you have access to, go to the [**DATA** tab](https://app.viam.com/data/view) where you can also filter by location, type of data, and more.
 
-To view captured data for a {{< glossary_tooltip term_id="part" text="machine part" >}}, click on the menu in the top right of its card or the menu in the machine resources list in the **Builder** menu and select **View captured data**.
+Or, you can navigate to the **DATA** tab from an individual resource, part, or machine's configuration card to automatically apply relevant filters:
 
-{{<imgproc src="/services/data/part-menu.png" resize="300x" declaredimensions=true alt="Machine menu with the options Rename, Restart part, View captured data, View setup instructions, View history, View debug configuration, and Delete machine">}}
+{{< tabs >}}
+{{% tab name="One resource" %}}
 
 To view captured data for a component or service, click on the menu in the top right of its card and select **View captured data**.
 
 ![Resource menu with the options Rename, Duplicate, View captured data, and Delete](/services/data/resource-menu.png)
 
-To view all the captured data you have access to, go to the [**DATA** tab](https://app.viam.com/data/view).
+{{% /tab %}}
+{{% tab name="Machine part" %}}
+
+To view captured data for a {{< glossary_tooltip term_id="part" text="machine part" >}}, click on the menu in the top right of its card or the menu in the machine resources list in the **Builder** menu and select **View captured data**.
+
+{{<imgproc src="/services/data/part-menu.png" resize="220x" declaredimensions=true alt="Machine menu with the options Rename, Restart part, View captured data, View setup instructions, View history, View debug configuration, and Delete machine">}}
+
+{{% /tab %}}
+{{% tab name="Whole machine" %}}
+
+To view captured data for a {{< glossary_tooltip term_id="machine" text="machine" >}} (including data from all parts of a multi-part machine), click on the data icon next to the **Save** button on the top right of the app.
+
+{{<imgproc src="/services/data/data-icon.png" resize="200x" declaredimensions=true alt="Data icon">}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< /expand >}}
+{{< expand "Step 5: (Optional) Pause sync" >}}
+
+You can pause cloud sync at any time by navigating to your machine's **CONFIGURE** tab and disabling **Syncing** for your [data management service](../).
+
+If you have captured data that you do not want to sync, delete the data on the machine before resuming cloud sync.
+To delete the data locally, `ssh` into your machine and delete the data in the directory where you capture data.
+
+{{< /expand >}}
+
+### Capture data selectively with filtering
+
+See the [Use filtering to collect and sync only certain images](/how-tos/image-data/#use-filtering-to-collect-and-sync-only-certain-images) guide.
```

Comment:
I think this may be useful as a call out during configuration instead. The same way you've dealt with pausing sync. 
Something like "Step 6: Too much data? Limit the captured data to interesting data"